plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -861,7 +866,8 @@ public void createMaterializedView(ConnectorSession session, SchemaTableName vie
                 .withStorage(storage -> storage.setStorageFormat(VIEW_STORAGE_FORMAT))
                 .withStorage(storage -> storage.setLocation(""))
                 .setViewOriginalText(Optional.of(
-                        encodeMaterializedViewData(fromConnectorMaterializedViewDefinition(definition))))
+                        encodeMaterializedViewData(
```

Comment:
MV was encoded created previously without `storage table`. Properties are also not really used (we derive them from table). I think we could remove `IcebergMaterializedViewDefinition` from `storageTable` and `properties` from `IcebergMaterializedViewDefinition`